cluster/gce/trusty/configure.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
```

Comment:
FWIW, I think mount-master-pd(), create-salt-master-auth() in configure-vm.sh are reusable as well. I'd like to have those in a seperate shell so all the distros can import.